### ISSUE TITLE
Front page ham menu is broken

### ISSUE THREAD

**Issue by smgriffin2** [imagej/imagej.github.io#48]
`index.md`'s hamburger menu is not fully appearing when activated. At first glance, it seems to be a `z-level` issue, due to a small grey box opening. This box ends when it hits the blue call-to-action section of the page. I've tried adjusting the `z-level`, as well tracking recent changes to `nav-bar`, `home.html` and relevant `main.css` code to ID potential issues, but have been unsuccessful in fixing the ham menu

**Comment by ctrueden** [comment]
The immediate issue is "fixed" by the following patch:

<details><summary>Patch</summary>

```diff
commit d7c81d149eecb24416a17fc3ad94edc4a8d9a941
Author: Curtis Rueden <ctrueden@wisc.edu>
Date:   Wed Apr 14 12:18:44 2021 -0500

    Move hamburger dropdown content to bottom of page
    
    This avoids an issue where later divs take precedence over earlier divs;
    see e.g. https://stackoverflow.com/a/11011083/1207769

diff --git a/_includes/footer b/_includes/footer
index b11e73621..5ec2d6231 100644
--- a/_includes/footer
+++ b/_includes/footer
@@ -7,3 +7,13 @@
             <li>&copy; ImageJ/FIJI</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
         </ul>
 </section>
+<div id="drop-menu" class="ham-drop-content">
+<a href="/">Home</a>
+<a href="/search" class="icon solid fa-search">Search</a>
+<a href="/downloads">Downloads</a>
+<a href="/learn">Learn</a>
+<a href="/develop">Develop</a>
+<a href="/news">News</a>
+<a href="/events">Events</a>
+<a href="/help">Help</a>
+</div>
diff --git a/_includes/nav-bar b/_includes/nav-bar
index 421a585a2..00bf60921 100644
--- a/_includes/nav-bar
+++ b/_includes/nav-bar
@@ -13,15 +13,5 @@
             <div class="bar1"></div>
             <div class="bar2"></div>
             <div class="bar3"></div>
-            <div id="drop-menu" class="ham-drop-content">
-                <a href="/">Home</a>
-                <a href="/search" class="icon solid fa-search">Search</a>
-                <a href="/downloads">Downloads</a>
-                <a href="/learn">Learn</a>
-                <a href="/develop">Develop</a>
-                <a href="/news">News</a>
-                <a href="/events">Events</a>
-                <a href="/help">Help</a>
-            </div>
         </div>
     </div>
diff --git a/assets/css/main.css b/assets/css/main.css
index a9bfaee39..5fb3151ba 100644
--- a/assets/css/main.css
+++ b/assets/css/main.css
@@ -1378,6 +1378,7 @@ input, select, textarea {
 			z-index: 1;
 			right: auto;
 			left: 0;
+			top: 50px;
 		}
 
 		.ham-drop-content a {
```

</details>

But the real problem is that the front page is showing a _different hamburger menu_ than what everything else is showing. So the behavior is different. With the patch above applied, one can see that the front page shows just the (old?) abbreviated nav bar on the front page, whereas other pages show the full main menu in there. So there must be some vestigial hamburger code that needs to removed, probably in `_layouts/home.html`, which is what the front page uses as its layout.

**Comment by ctrueden** [comment]
Addressed by 20c1974b73f50bf6e36c867aa1baa7891b312b9c.
